closed/OpenJ9.gmk

```diff
@@ -224,6 +224,13 @@ else
   FEATURE_SED_SCRIPT += $(call SedDisable,opt_jitserver)
 endif
 
+# Adjust InlineTypes enablement flags.
+ifeq (true,$(OPENJ9_ENABLE_INLINETYPES))
+  FEATURE_SED_SCRIPT += $(call SedEnable,opt_valhallaValueTypes)
+else
+  FEATURE_SED_SCRIPT += $(call SedDisable,opt_valhallaValueTypes)
+endif
```

Comment:
This needs to be in `SPEC_SED_SCRIPT` (not `FEATURE_SED_SCRIPT`) as it is `*.spec` files that need to be adjusted.